internal/input/validate_test.go

```diff
@@ -119,14 +119,14 @@ func Test_ValidatePipeline(t *testing.T) {
 			name:    "validation succeeds with json input",
 			fpath:   "{\"json\": 1}",
 			err:     nil,
-			output:  &output.Output{PolicyCheck: []evaluator.Outcome{}},
+			output:  &output.Output{PolicyCheck: []evaluator.Outcome(nil)},
 			defFunc: mockNewPipelineDefinitionFile,
 		},
 		{
 			name:    "validation succeeds with yaml input",
 			fpath:   "kind: task",
 			err:     nil,
-			output:  &output.Output{PolicyCheck: []evaluator.Outcome{}},
+			output:  &output.Output{PolicyCheck: []evaluator.Outcome(nil)},
```

Comment:
I'm curious about this diff, is it just a style change?

Reply:
It was necessary to allow tests to pass. AFAICT it's just a different type of "nothing" since before it was just an empty list.